package.json

```diff
@@ -19,14 +19,15 @@
   "license": "UNLICENSED",
   "private": false,
   "dependencies": {
-    "asar": "^3.0.3",
-    "chalk": "^4.0.0",
-    "globby": "^11.0.1",
-    "prompts": "^2.3.2",
-    "semver-sort": "^0.0.4"
+    "asar": "3.0.3",
```

Comment:
👍 for absolute versioning, but the packages are installed with yarn, so the package-lock.json changes actually be yarn.lock changes.